README.md

```diff
@@ -0,0 +1,12 @@
+# PeloTech - Foundation
```

Comment:
Looks good, could we merge this with the existing one in the docs folders? We through them there since that's where mkdocs defaults too and github honors the location as well :)